db/migrate/20210426150053_add_la_funded_place_urn_to_schools.rb

```diff
@@ -0,0 +1,6 @@
+class AddLaFundedPlaceUrnToSchools < ActiveRecord::Migration[6.1]
+  def change
+    add_column :schools, :provision_urn, :string, index: true
```

Comment:
this `index: true` does not look to be correct. because if you look at the schema no index is added. maybe use `add_index` as an easy fix?

Reply:
Great spot, I've changed this to use `add_index` as suggested